react/tic-tac-toe/src/app/screens/Login/components/CustomInput/styles.module.scss

```diff
@@ -0,0 +1,5 @@
+.row {
+  display: flex;
+  flex-direction: row;
+}
+  
```

Comment:
Leave a white line here

Also, I'd add this in your `src/scss/layout.scss` since this is a reusable CSS class. Finally, you don't need to add the `.module` extension in order for this to work.